tools/static-assets/server/runtime.js

```diff
@@ -47,15 +47,15 @@ module.exports = function enable ({ cachePath, createLoader = true } = {}) {
   };
 
   const reifyVersion = require("@meteorjs/reify/package.json").version;
-  const reifyBabelParse = require("@meteorjs/reify/lib/parsers/babel").parse;
+  const reifyAcornParse = require("@meteorjs/reify/lib/parsers/acorn").parse;
   const reifyCompile = require("@meteorjs/reify/lib/compiler").compile;
 
   function compileContent (content) {
     let identical = true;
 
     try {
       const result = reifyCompile(content, {
-        parse: reifyBabelParse,
+        parse: reifyAcornParse,
```

Comment:
Using `reifyAcornParse` instead of `reifyBabelParse` in places used has improved further. The profile in the description is not updated on this.